Makefile

```diff
@@ -1477,16 +1467,21 @@ runtime/%.i.o: runtime/%.S
 runtime/%_libasmrunpic.o: runtime/%.S
 	$(V_ASM)$(ASPP) $(OC_ASPPFLAGS) $(SHAREDLIB_CFLAGS) -o $@ $<
 
-runtime/domain_state.inc: runtime/caml/domain_state.tbl
-	$(V_GEN)$(CPP) $< > $@
+runtime/domain_state64.inc: \
```

Comment:
This change and the parts which follow appear to be a rebasing error of some kind?